android-components/.buildconfig.yml

```diff
@@ -190,7 +190,6 @@ projects:
     upstream_dependencies:
     - browser-state
     - concept-engine
-    - support-ktx
```

Comment:
@jonalmeida I'm assuming this safe to remove now? This was added due to the transitive dependency build issue we were seeing https://github.com/mozilla-mobile/firefox-android/pull/948/files#diff-07bb9c7b5be613874316fa2adac2ac55725f5fd9dcd4f17c958a73bc1c8c44ef

Reply:
I think so too.

Maybe after rebasing on top of https://github.com/mozilla-mobile/firefox-android/pull/1090 the build errors might go away.